math/unuran/test/unuranSampler.cxx

```diff
@@ -0,0 +1,234 @@
+// test TUnuran Sampler class using t he DistSampler interface 
+#include "gtest/gtest.h"
+
+#include "Math/DistSampler.h"
+#include "Math/DistSamplerOptions.h"
+#include "Math/Factory.h"
+#include "Math/DistFunc.h"
+#include "Math/Functor.h"
+#include "TH1.h"
+#include "TH2.h"
+
+#include "ROOT/RMakeUnique.hxx"
+
+using namespace ROOT::Math; 
+
+std::unique_ptr<TH1D> FillHisto1D( DistSampler & s) {
+    // create histogram using automatic binning
+    auto h1 = std::make_unique<TH1D>("h1","h1",100, 1, 0);  
+    const int nevt = 10000; 
+    for (int i = 0; i < nevt; i++) {
+        h1->Fill( s.Sample1D());
+    }
+    return h1;
+}
+
+// test using Unuran string API
+TEST(OneDim, StringAPI)
+{
+    std::unique_ptr<DistSampler> sampler(Factory::CreateDistSampler("Unuran"));
+
+    bool ret = sampler->Init("distr = normal(3.,0.75); domain = (0,6) & method = tdr; c = 0");
+    EXPECT_EQ(ret, true);
+    if (!ret) return;
+
+    sampler->SetMode(0.75);
```

Comment:
I don't understand why the Mode is set to `0.75` here, should it not be 3.0 because the mean of a normal distribution is also the mean?